apps/omg_watcher/test/omg_watcher/integration/in_flight_exit_test.exs

```diff
@@ -39,129 +37,6 @@ defmodule OMG.Watcher.Integration.InFlightExitTest do
   # bumping the timeout to three minutes for the tests here, as they do a lot of transactions to Ethereum to test
   @moduletag timeout: 180_000
 
-  @tag fixtures: [:in_beam_watcher, :alice, :bob, :mix_based_child_chain, :alice_deposits]
-  test "piggyback in flight exit", %{alice: alice, bob: bob, alice_deposits: {alice_deposit, _}} do
-    {:ok, _} = DevHelper.import_unlock_fund(bob)
-    bob_deposit = DepositHelper.deposit_to_child_chain(bob.addr, 10)
-
-    submitted_tx =
```

Comment:
question @pdobacz 
I've directly mapped this submitted_tx into:
```
# inputs
    alice_deposit_utxo = hd(alice_utxos)

    alice_deposit_input = %ExPlasma.Utxo{
      blknum: alice_deposit_utxo["blknum"],
      currency: Currency.ether(),
      oindex: 0,
      txindex: 0,
      output_type: 1,
      owner: alice_address
    }

    bob_deposit_utxo = hd(bob_utxos)

    bob_deposit_input = %ExPlasma.Utxo{
      blknum: bob_deposit_utxo["blknum"],
      currency: Currency.ether(),
      oindex: 0,
      txindex: 0,
      output_type: 1,
      owner: bob_address
    }

    # outputs
    alice_output = %ExPlasma.Utxo{
      currency: Currency.ether(),
      owner: alice_address,
      amount: alice_child_chain_balance - Currency.to_wei(5)
    }

    bob_output = %ExPlasma.Utxo{
      currency: Currency.ether(),
      owner: bob_address,
      amount: amount + bob_child_chain_balance
    }

    transaction = %Payment{inputs: [alice_deposit_input, bob_deposit_input], outputs: [alice_output, bob_output]}
```

the question I have here is ... I first thought Alice is sending 5 ETH to Bob, but I wonder why did we provide Bob deposit information as inputs? What sort of a scenario did this get us into so that later IFE made sense?

Reply:
Hm, indeed, if this test case focuses only on the piggyback invalidity, 1input/1output txs would suffice. Possibly an oversight after these tests being split between piggyback-focused and canonicity-focused cases.

FWIW, for canonicity-focused cases you could also go by just 1in/1out, but the most "prominent" example use case is when 2 actors co-sign a transaction and one of them is malicious, so this is probably why 2 inputs is here.

Does this clarify this a bit?